src/main/java/nextstep/subway/applicaion/exception/NotFoundException.java

```diff
@@ -0,0 +1,15 @@
+package nextstep.subway.applicaion.exception;
+
+public class NotFoundException extends RuntimeException{
+
+    public NotFoundException() {
+    }
+
+    public NotFoundException(String message) {
+        super(message);
+    }
```

Comment:
코멘트에서 하신 질문은 이렇게 하시면 됩니다 😅 
변경하신 방법은 다시 매직 스트링이 예외를 사용하는 클래스로 이동하게 되었네요 😢 

```suggestion
private static String m = "해당하는 대상을 찾을 수 없습니다. id : %s";
public NotFoundException(Long id) {
    super(String.format(m, id));
}
```